bson/_cbsonmodule.c

```diff
@@ -599,8 +599,7 @@ int cbson_convert_type_registry(PyObject* registry_obj, type_registry_t* registr
  * Return 1 on success. options->document_class is a new reference.
  * Return 0 on failure.
  */
-int convert_codec_options(PyObject* self, PyObject* options_obj, void* p) {
-    codec_options_t* options = (codec_options_t*)p;
+int convert_codec_options(PyObject* self, PyObject* options_obj, codec_options_t* options) {
```

Comment:
To clarify--what was the reason for the `options` parameter being a generic void pointer before this change?